src/librustc/hir/map/mod.rs

```diff
@@ -68,7 +68,7 @@ pub enum Node<'hir> {
     NodeStructCtor(&'hir VariantData),
 
     NodeLifetime(&'hir Lifetime),
-    NodeTyParam(&'hir TyParam),
+    NodeGenericParam(&'hir GenericParam),
```

Comment:
Do you need to remove `NodeLifetime`? Or I guess that would remain in use for uses of lifetimes, but not definitions?

Reply:
Yes, as far as I can tell it's only used for lifetimes now.